tests/resolvers/Chat/admins.spec.ts

```diff
@@ -0,0 +1,40 @@
+import "dotenv/config";
+import { admins as adminsResolver } from "../../../src/resolvers/Chat/admins";
+import { connect, disconnect } from "../../helpers/db";
+import type mongoose from "mongoose";
+import { User } from "../../../src/models";
+import { afterAll, beforeAll, describe, expect, it } from "vitest";
+import type { TestChatType } from "../../helpers/chat";
+import { createTestChatMessage } from "../../helpers/chat";
+
+let testChat: TestChatType;
+let MONGOOSE_INSTANCE: typeof mongoose;
+
+beforeAll(async () => {
+  MONGOOSE_INSTANCE = await connect();
+  const userOrgChat = await createTestChatMessage();
+  testChat = userOrgChat[2];
+});
+
+afterAll(async () => {
+  await disconnect(MONGOOSE_INSTANCE);
+});
+
+describe("resolvers -> Chat -> admins", () => {
+  it(`returns user object for parent.users`, async () => {
+    const parent = testChat?.toObject();
+    if (!parent) {
+      throw new Error("Parent object is undefined.");
+    }
+
+    const usersPayload = await adminsResolver?.(parent, {}, {});
```

Comment:
The `adminsResolver?.(parent, {}, {})` uses optional chaining (`?.`), which is unnecessary if you're confident that the `adminsResolver` will always be defined. If you want to safeguard against undefined resolvers, it’s better to throw an error early on:

```javascript
if (!adminsResolver) {
  throw new Error("adminsResolver is not defined.");
}
const usersPayload = await adminsResolver(parent, {}, {});
```